fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/ScanComparisons.java

```diff
@@ -206,6 +206,11 @@ public Builder() {
             super(new ArrayList<>(), new HashSet<>());
         }
 
+        public void clear() {
+            this.equalityComparisons.clear();
+            this.inequalityComparisons.clear();
+        }
```

Comment:
Should this return `this` for consistency?

Reply:
done.